src/components/CardList/CardList.js

```diff
@@ -4,13 +4,37 @@ import Actions from '../CardList/Actions';
 import './CardList.css';
 import Page404 from '../Page404/Page404';
 import {Route} from 'react-router-dom';
-
+import axios from 'axios';
 import { connect } from 'react-redux';
+import { onFillCardData, onError} from '../../store/actions';
+
+class CardList extends Component {
 
-class CardList extends Component {  
+  fillCards = () => {
+    axios.get('https://raw.githubusercontent.com/BrunnerLivio/PokemonDataGraber/master/output.json')
+        .then(response => {
+            const cardsList = response.data.slice(0, 15).map(card => {
+                return {
+                    id: card['Number'],
+                    headerText: card['Name'],
+                    bodyText: card['About']
+                };
+            });
```

Comment:
```suggestion
            const cardsList = response.data.slice(0, 15).map(card => ({
                    id: card.Number,
                    headerText: card.Name,
                    bodyText: card.About
            }));
```
some QA systems might complain

Reply:
done